server/spec/subscription_resource_spec.rb

```diff
@@ -37,4 +37,36 @@
 
       @cp.activate_subscription(consumer.uuid, "mail", "locale")
   end
+
+  it 'subscriptions derived/provided products should return empty array when null' do
+
+      # Product does not have derived/provided products
+      pool = create_pool_and_subscription(@owner['key'], @some_product.id, 2,
```

Comment:
Same as the 4.0 PR: Please use the non-deprecated methods for creating and deleting pools

Reply:
ack. It worked now.